compose/__init__.py

```diff
@@ -1,4 +1,8 @@
 from __future__ import unicode_literals
 from .service import Service  # noqa:flake8
 
+from collections import namedtuple
+
 __version__ = '1.3.0dev'
+
+VolumeFrom = namedtuple("VolumeFrom", 'source mount_option')
```

Comment:
minor point, but this is only used in one module right now (outside of tests), so it would be good to keep it local to that module.  

It's good to keep `__init__.py` as small as possible